### DISCUSSION TITLE
SBUS RECEIVER PROBLEM - Issues with default / failsafe values (885 and 1675)

### DISCUSSION THREAD

**Discussion by dzaro-dev** [iNavFlight/inav#7602] [Q&A]
## Current Behavior
After updating to INAV 3.0.2, I noticed the following behaviour / issue and I have been having issues working out what is happening. The configuration has been started from scratch and nothing has been misconfigured by using DUMP or even DIFF ALL settings from the prvious version 2.6.1.

For information, I am using the following devices:

- Futaba T14SG transmitter 
- Futaba R7008SB SBUS receiver.
- Matek F765 FC

I have adjusted the MAX and MIN endpoints/range on my transmitter for all channels in order to obtain output values of 1000 to 2000, with a mid at 1500.

**In version 2.6.1:**
Notice the values when plugging in the FC via USB only and not powering on the Futaba transmittter: All of the channles are set to 1500, i.e. mid-stick, and the failsafe icon lights up RED immediately after connecting the FC to a USB port. (In my opinion, that is what it should display.)

![9  Receiver - 1](https://user-images.githubusercontent.com/23706061/138558793-6c7043fd-12a2-4a41-a277-e5d4ba81dade.PNG)

**In version 3.0.2:**
When configuring everything exactly the same way manually and leaving the transmitter off, after plugging in the USB the throttle channel is set automatically set to 885 and channel 5 is set to 1675. This is very stange because, as mentioned before, the minimum out for any channel was manually set so that it equates to 1000. The same happens after turning ON the transmitter and then turning it back OFF again. These two channels go back to these random values (well, at least random to me). I also noticed that the failsafe sometimes comes on, but sometimes does not and this is also quite unexpected.

![STRANGE FAULT](https://user-images.githubusercontent.com/23706061/138559262-b890330b-4758-49ff-ba40-9abcc4131400.PNG)

## Expected behavior
As I have started using INAV from version 3.0.2, I don't know which one of these is the correct representation of the channels in failsafe mode. The Futaba T14SG manual also does not offer a lot of information regarding failsafe settings and it definitely does not show the throttle value below the absolute minimum of 1000 when moving the sticks with the transmitter ON.

## Suggested solution(s)
I have no idea but maybe having a look at the old SBUS code and seeing if there are any differences there perhaps. 

## Additional context
I have reflashed the FC (just to be on the safe side), but there seems to be some kind of a chnage in how the signal-loss (in this case lack of signal altogether) is being processed and recognised by the FC. I reflashed the old 2.6.1 firmware and installed the settings and configurations from the latest 2.6.1 dump-file I created. The setting are definitely back to mid-stick, i.e. 1500, as their default, when there is no connection to the SBUS receiver.

---
# version
# INAV/MATEKF765 3.0.2 Sep 15 2021 / 10:27:24 (66011184)
# GCC-9.3.1 20200408 (release)

**DIFF ALL and DUMP files of each version are below:**

**Version 2.6.1**
[INAV_cli_DIFF_ALL_FINAL.txt](https://github.com/iNavFlight/inav/files/7403136/INAV_cli_DIFF_ALL_FINAL.txt)
[INAV_cli_DUMP_FINAL.txt](https://github.com/iNavFlight/inav/files/7403137/INAV_cli_DUMP_FINAL.txt)

Version 3.0.2
[INAV_cli_DIFF_ALL_MATCHED.txt](https://github.com/iNavFlight/inav/files/7403238/INAV_cli_DIFF_ALL_MATCHED.txt)
[INAV_cli_DUMP_MATCHED.txt](https://github.com/iNavFlight/inav/files/7403239/INAV_cli_DUMP_MATCHED.txt)

**Comment by rwindsorjr** [comment]
all of your limits are off. 1000  min to 2000 max. need to set them if you can . I have a TX16S and uses Open TX. I have to set limits in the outputs page .  -97.5 on low end and 97.5 on the high on all channels. You settings may vary .
